From 40a6fd7231df123fec042d198be33830b3f33806 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Wed, 20 Aug 2025 08:33:26 +0200
Subject: [PATCH 54/55] ubox: silence-warnings

Suppresses non-critical kmodloader warning messages during module loading to reduce log noise.

Signed-off-by: John Crispin <john@phrozen.org>
---
 .../ubox/patches/100-silence-warnings.patch   | 20 +++++++++++++++++++
 1 file changed, 20 insertions(+)
 create mode 100644 package/system/ubox/patches/100-silence-warnings.patch

diff --git a/package/system/ubox/patches/100-silence-warnings.patch b/package/system/ubox/patches/100-silence-warnings.patch
new file mode 100644
index 0000000000..aebe05a21c
--- /dev/null
+++ b/package/system/ubox/patches/100-silence-warnings.patch
@@ -0,0 +1,20 @@
+--- a/kmodloader.c
++++ b/kmodloader.c
+@@ -417,7 +417,7 @@ static char *mmap_modinfo(const char *mo
+ 
+ 	map = mmap(NULL, s->st_size, PROT_READ, MAP_PRIVATE, fd, 0);
+ 	if (map == MAP_FAILED) {
+-		ULOG_ERR("failed to mmap %s\n", mpath);
++		//ULOG_ERR("failed to mmap %s\n", mpath);
+ 		goto out;
+ 	}
+ 
+@@ -547,7 +547,7 @@ static int scan_builtin_modules(void)
+ 		}
+ 		m = get_module_info(NULL, name);
+ 		if (!m) {
+-			ULOG_ERR("failed to find info for builtin module %s\n", name);
++			//ULOG_ERR("failed to find info for builtin module %s\n", name);
+ 			goto err;
+ 		}
+ 	}
-- 
2.34.1

